### PR TITLE
add missing dependencies in OpenJPEG easyconfig (cfr. #7216)

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

File: easybuild/easyconfigs/o/OpenJPEG/OpenJPEG-2.3.0-GCCcore-7.3.0.eb [hunk 1]
```diff
@@ -19,8 +19,8 @@ toolchain = {'name': 'GCCcore', 'version': '7.3.0'}
 toolchainopts = {'pic': True}
 
 source_urls = ['https://github.com/uclouvain/openjpeg/archive']
-sources = ['v%(version)s.zip']
-checksums = ['e0bd4ee9ab6e4f3a581fca5e01fbfb45a8ab9062baeddd1211741531b53fe07d']
+sources = ['v%(version)s.tar.gz']
+checksums = ['3dc787c1bb6023ba846c2a0d9b1f6e179f1cd255172bde9eb75b01f1e6c7d71a']
```
Comment (boegel) [inline easybuild/easyconfigs/o/OpenJPEG/OpenJPEG-2.3.0-GCCcore-7.3.0.eb]: This was only changed to be in sync with #7216

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2002.delcatty.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/fb1109b8afe08a2e3dac0a5c83a2c70a for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3100.skitty.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/891d5757c605abeab44c1cd6b2375e61 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2638.swalot.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/a9e3d7fda91bffc5d2cd54256448c838 for a full test report.

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
grc-cluster1 - Linux centos 6.10, Intel(R) Xeon(R) CPU E5-2640 0 @ 2.50GHz, Python 2.7.14
See https://gist.github.com/4ce8cd48c5fe7577e8c148e47c7212c8 for a full test report.

**Comment by migueldiascosta** [comment]
Going in, thanks @boegel!

**Pull request merged**